tiledb/bioimg/converters/ome_zarr.py

```diff
@@ -2,14 +2,23 @@
 
 import json
 import logging
+import warnings
 from typing import Any, Dict, List, Mapping, Optional, Sequence, Tuple, cast
 
 import numpy
 import numpy as np
-import zarr
-from numcodecs import Blosc
-from ome_zarr.reader import OMERO, Multiscales, Reader, ZarrLocation
-from ome_zarr.writer import write_multiscale
+
+try:
+    import zarr
+    from numcodecs import Blosc
```

Comment:
Should the numcodecs import be outside of this try block? If it fails it will warn about the wrong module.

Reply:
Context-free that is true. However the `numcodecs` is a direct dependency of `zarr` and installing the latter installs the former. Thank you for this comment though cause based on this one I will change this import from `from numcodecs import Blosc` to `from zarr.codecs import Blosc` where they just wrap the `numcodecs.Blosc` importation. This way it will be cleaner.

```python

# site-packages/zarr/codecs.py
# flake8: noqa
from numcodecs import *
from numcodecs import get_codec, Blosc, Pickle, Zlib, Zstd, Delta, AsType, BZ2
from numcodecs.registry import codec_registry
```